### PR TITLE
20210601 grants update

### PR DESCRIPTION
**Pull request by mikeosborne**
Added 3rd party software distribution apprenticeship already completed by Raghu

### CODE REVIEW

**Comment by mikeosborne** [comment]
@jalehman - the vane documentation bounty has a couple of missing links marked with "TODO"

File: content/grants/bounty-calendar-ui.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
++++
+title = "Calendar UI"
+date = 2021-01-01
+[taxonomies]
+grant_type = ["bounties"]
```
Comment (jalehman) [inline content/grants/bounty-calendar-ui.md:5]: This is a proposal, not a bounty.

File: content/grants/bounty-calendar-ui.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+grant_category = ["App Dev: Other"]
+[extra]
+image = ""
+description = "Develop a UI for improved access to the Calendar Gall app"
+reward = 5
```
Comment (jalehman) [inline content/grants/bounty-calendar-ui.md:10]: This should be 3, not 5

File: content/grants/bounty-calendar-ui.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+assignee = "~fabnev-hinmur"
+id = "B0014"
+completed = true
+link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0014&prefill_Grant+Name=ucal%20Google%20and%20Outlook%20Integration"
```
Comment (jalehman) [inline content/grants/bounty-calendar-ui.md:14]: This isn't necessary

File: content/grants/bounty-calendar-ui.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+date = 2021-01-01
+[taxonomies]
+grant_type = ["bounties"]
+grant_category = ["App Dev: Other"]
+[extra]
+image = ""
+description = "Develop a UI for improved access to the Calendar Gall app"
+reward = 5
+assignee = "~fabnev-hinmur"
+id = "B0014"
```
Comment (jalehman) [inline content/grants/bounty-calendar-ui.md:12]: I think the ID maps to something @tylershuster implemented for the old grants site. Might want to double check the template to make sure this doesn't create any weird links.

File: content/grants/bounty-calendar-ui.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+9. Toggle calendar display
+
+## Future features
+
+There will be additional UI features necessary to allow users to interact with other ships' calendars. Specifically, retrieving events that a user is invited to, RSVPing to events, subscribing to other ships' calendars, modifying other calendars with requisite permissions, and perhaps requesting additional permissions on other calendars. Many of these features are in progress but still require significant development and testing.
+
+
+## Milestones
+
+Completion of all 9 features: 5 stars
```
Comment (jalehman) [inline content/grants/bounty-calendar-ui.md:53]: See above about reward.

**Review by jalehman — CHANGES_REQUESTED** [review]
Just a few changes about the calendar UI.

**Comment by jalehman** [comment]
I see two calendar UI grants now: image

**Comment by jalehman** [comment]
And they both still seem to be listed under bounties for some reason.

**Comment by mikeosborne** [comment]
commits folded into #639

**Pull request closed without merging**